addons/account/models/account_move.py

```diff
@@ -130,6 +130,25 @@ def _get_default_invoice_incoterm(self):
         ''' Get the default incoterm for invoice. '''
         return self.env.company.incoterm_id
 
+    @api.model
+    def _company_terms(self, company):
+        company_terms_note = company.terms_type != 'html'
```

Comment:
This doesn't sound "boolean". I was expecting the field to hold a string, given its name.